packages/shared/utils/is.ts

```diff
@@ -23,3 +23,4 @@ export const rand = (min: number, max: number) => {
   max = Math.floor(max)
   return Math.floor(Math.random() * (max - min + 1)) + min
 }
+export const isIOS = /* #__PURE__ */ isClient && window?.navigator?.userAgent && /iP(ad|hone|od)/.test(window.navigator.userAgent)
```

Comment:
Note. that the judgment condition has been changed from `navigator.platform` to `navigator.userAgent` 

`navigator.platform` is deprecated.

Reply:
Sure, the new implementation has avoid to use `navigator.platform`